keyboards/sawnsprojects/satxri6key/keymaps/via/config.h

```diff
@@ -0,0 +1,19 @@
+#ifdef RGB_DI_PIN
```

Comment:
This should be moved into the keyboard's config.h file.

Otherwise, this file needs both a license header and an include guard (`#pragma once`)